subt/doc/virtual-howto.md

```diff
@@ -0,0 +1,61 @@
+# SubT Challenge - Virtual Track HowTo
+
+Here are instructions how to build, run locally and deploy to AWS "robotika" solution docker.
+It is expected that you already have docker installed and your PC has compatible NVidia GPU.
+For official instructions how to start with Docker please look at:
+  https://bitbucket.org/osrf/subt/wiki/tutorials/SystemSetupDockerInstall
+
+
+## Prepare OSGAR build environment
+The convenience scripts for build and run are merged into OSGAR:
+```
+git clone https://github.com/robotika/osgar.git .
+cd ~/osgar/subt/docker
```

Comment:
"osgar/subt" directory does not exists after the git clone above. The sequence should be prefixed with "mkdir osgar ; cd osgar"

Do we have to assume that everything is in the ${HOME} directory? It is a minor issue, but it means that a reader who uses a different directory structure cannot simply copy-paste the commands.

Reply:
I am not sure if to enforce all installs into the same place - subt uses for example "~/subt_ws/. Now everything expects that you start section at osgar root directory.